post-meta-inspector.php

```diff
@@ -1,38 +1,66 @@
-<?php
+<?php // phpcs:ignore
```

Comment:
FWIW, this comment only prevents `phpcs` from flagging that this file should be named `class-post-meta-inspector.php`.